bluechi.spec.in

```diff
@@ -203,7 +208,11 @@ API description and manually written code to simplify recurring tasks.
 %autosetup
 
 %build
+%if %{with_coverage}
 %meson -Dapi_bus=system
+%else
+%meson -Dapi_bus=system -Dwith_coverage=true
+%endif
```

Comment:
Shouldn't this be reversed?
```suggestion
%if %{with_coverage}
%meson -Dapi_bus=system -Dwith_coverage=true
%else
%meson -Dapi_bus=system
%endif
```
If `with_coverage` is true, then add the meson config option.